shared/wallets/wallet-list/index.tsx

```diff
@@ -82,12 +81,13 @@ const WhatIsStellar = (props: {onWhatIsStellar: () => void}) => (
 
 export type Props = {
   accountIDs: Array<AccountID>
+  airdropIsLive: boolean
   airdropSelected: boolean
   style?: Styles.StylesCrossPlatform
   loading: boolean
   inAirdrop: boolean
   onAddNew: () => void
-  onJoinAirdrop: () => void
+  onJoinAirdrop: () => void | null
```

Comment:
I think you want `(()=>void)|null`. Otherwise it's `() => (void | null)` IIRC. Seems like this can also be `onJoinAirdrop?: ...`

Reply:
Thanks, fixed.